Sources/Purchasing/ProductsManager.swift

```diff
@@ -119,9 +121,11 @@ private extension ProductsManager {
             productsFetcherSK1.products(withIdentifiers: removedProductIdentifiers, completion: { result in
                 switch result {
                 case.success:
-                    Logger.debug(Strings.storeKit.skproductsrequest_finished)
+                    // this one is duplicate, though other in ProductsFetcherSK1 is rcSuccess
```

Comment:
options i see are:
*. remove duplication -- need to look into hierarchy of logging, when would rcSuccess happen vs debug?
* ensure sk2 has the same duplication

Reply:
I don't think we've looked into these logs very much in detail, so this refactor is great 👍🏻 

`rcSuccess` is just a success `LogIntent`, but still `LogLevel.debug`. I think that's good for these success cases.